[2025-03-31T21:32:48.208-0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-31T21:32:48.212-0300] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: employee_changes_report.extract_from_mysql scheduled__2025-03-01T00:00:00+00:00 [queued]>
[2025-03-31T21:32:48.214-0300] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: employee_changes_report.extract_from_mysql scheduled__2025-03-01T00:00:00+00:00 [queued]>
[2025-03-31T21:32:48.214-0300] {taskinstance.py:2303} INFO - Starting attempt 5 of 6
[2025-03-31T21:32:48.221-0300] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): extract_from_mysql> on 2025-03-01 00:00:00+00:00
[2025-03-31T21:32:48.224-0300] {standard_task_runner.py:63} INFO - Started process 12224 to run task
[2025-03-31T21:32:48.229-0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'employee_changes_report', 'extract_from_mysql', 'scheduled__2025-03-01T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/employee_changes_report_dag.py', '--cfg-path', '/var/folders/pw/ybzcyby54d787c124gv_5tjw0000gn/T/tmp52w1_cex']
[2025-03-31T21:32:48.230-0300] {standard_task_runner.py:91} INFO - Job 21: Subtask extract_from_mysql
[2025-03-31T21:32:48.245-0300] {task_command.py:426} INFO - Running <TaskInstance: employee_changes_report.extract_from_mysql scheduled__2025-03-01T00:00:00+00:00 [running]> on host ordenador-portatil-de-user-2.local
[2025-03-31T21:32:48.625-0300] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='employee_changes_report' AIRFLOW_CTX_TASK_ID='extract_from_mysql' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-01T00:00:00+00:00'
[2025-03-31T21:32:48.626-0300] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-31T21:32:48.647-0300] {base.py:84} INFO - Using connection ID 'mysql_employees' for task execution.
[2025-03-31T21:32:48.689-0300] {logging_mixin.py:188} WARNING - /Users/user/Documents/repo test eng/env/lib/python3.10/site-packages/airflow/providers/common/sql/hooks/sql.py:400 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-03-31T21:32:55.322-0300] {python.py:237} INFO - Done. Returned value was: None
[2025-03-31T21:32:55.322-0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-31T21:32:55.324-0300] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=employee_changes_report, task_id=extract_from_mysql, execution_date=20250301T000000, start_date=20250401T003248, end_date=20250401T003255
[2025-03-31T21:32:55.364-0300] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-03-31T21:32:55.370-0300] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-31T21:32:55.370-0300] {local_task_job_runner.py:222} INFO - ::endgroup::
